package rec.audio;

message AudioDeviceSetupProto {
  optional string output_device_name = 1;
  optional string input_device_name = 2;
  optional double sample_rate = 3;
  optional uint64 buffer_size = 4;
  optional bytes input_channels = 5;
  optional bool use_default_input_channels = 6;
  optional bytes output_channels = 7;
  optional bool use_default_output_channels = 8;
};